modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java

```diff
@@ -277,14 +277,14 @@ public interface Transaction extends AutoCloseable, IgniteAsyncSupport {
     public IgniteFuture<Void> rollbackAsync() throws IgniteException;
 
     /**
-     * Resume a transaction if it was previously suspended. <strong>Supported only for optimistic transactions.</strong>
+     * Resume a transaction if it was previously suspended. <strong>Now, It supports pessimistic transactions and optimistic transactions.</strong>
```

Comment:
Might be better to delete - <strong>Supported only for optimistic transactions.</strong>?